extensions/wikia/UserLogin/js/UserBaseAjaxForm.js

```diff
@@ -44,14 +43,6 @@
 	};
 
 	/**
-	 * Bind form events
-	 */
-	UserBaseAjaxForm.prototype.bindEvents = function () {
-		this.form.on('submit', this.submitLogin.bind(this));
```

Comment:
Yeah, good catch. Was a left over from the larger cleanup that I undid, I missed restoring this. This extension is only used on devboxes and would still work (you'd just go to Special:UserLogin on an error rather than it being displayed in the form) but I'll restore it.